book/chapters/chapter11/large-scale_benchmarking.qmd

```diff
@@ -181,7 +186,7 @@ path_adult_tasks = file.path("openml", "manual", "adult_tasks.rds")
 
 ```{r, eval = !file.exists(path_adult_tasks)}
 # first 5 tasks making use of the adult data
-adult_tasks = list_oml_tasks(data_id = 1590, limit = 5)
+adult_tasks = list_oml_tasks(data_id = 1590)
```

Comment:
We are deliberately suppressing a lot of output to save space in the book.

```suggestion
adult_tasks = list_oml_tasks(data_id = 1590, limit = 5)
```

Reply:
This was super confusing because the "randomly picked id" was not part of the returned sample of 5 data sets. The output is still quite concise as only the ids are printed, not the table.